scripts/freshvenvs.py

```diff
@@ -8,16 +8,35 @@
 import pathlib
 import sys
 import typing
+from typing import Optional
+
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.resolve()))
 
 from packaging.version import Version
 from pip import _internal
 
+import riotfile
 
-sys.path.append(str(pathlib.Path(__file__).parent.parent.resolve()))
 
 CONTRIB_ROOT = pathlib.Path("ddtrace/contrib")
 
 
+# Venv suite name -> integration name
```

Comment:
Is this comment accurate? It's probably better to name the variable such that this comment becomes redundant.